### PR TITLE
Add Lenovo ThinkPad X280 profile

### PR DESCRIPTION
**Pull request by mishudark**

### CODE REVIEW

**Comment by Mic92** [comment]
Please also add a link in https://github.com/NixOS/nixos-hardware/blob/master/README.md

File: lenovo/thinkpad/x280/default.nix [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  imports = [
+    ../.
+    ../../../common/cpu/intel
+    ../../../../common/pc/laptop/acpi_call.nix
```
Comment (Mic92) [inline lenovo/thinkpad/x280/default.nix]: This one might make also sense:


```
../../common/pc/laptop/ssd
```

It seems like they sell SSDs for all devices.

Comment (mishudark) [reply lenovo/thinkpad/x280/default.nix]: you are right, fixed

**Comment by mishudark** [comment]
Link added to Readme.md

**Pull request merged**